proto/buf.work.yaml

```diff
@@ -1,3 +1,3 @@
 version: v1
 directories:
-  - proto
+  - astria
```

Comment:
should we name the top level folder `src` so its `src/astria` will that break the module naming stuff?

Reply:
The issue that this repetition addresses is potentially vendoring other protobufs. Here for example is what penumbra is doing: https://github.com/penumbra-zone/penumbra/tree/main/proto

They have `penumbra` for their own protobuf, but also `rust-vendored` and another.

Buf in their example has the `paymentapis` and `petapis`: https://buf.build/docs/reference/workspaces#configuration

I think the right move here is to get rid of the workspace until we actually make use of it. I will change this.